hack/build/init-buildx.sh

```diff
@@ -21,7 +21,7 @@ export DOCKER_CLI_EXPERIMENTAL=enabled
 # We can skip setup if the current builder already has multi-arch
 # AND if it isn't the docker driver, which doesn't work
 current_builder="$(docker buildx inspect)"
-# linux/amd64, linux/arm64, linux/riscv64, linux/ppc64le, linux/s390x, linux/386, linux/arm/v7, linux/arm/v6
+# linux/amd64, linux/arm64, linux/riscv64, linux/ppc64le, linux/s390x, linux/386, linux/arm/v7, linux/arm/v6, linux/mips64le
```

Comment:
note: this is just a comment showing an example of what we're looking for, this comment value isn't actually used at all.
we're just detecting the difference between a builder that has qemu binfmt_misc detected (and therefore N>1 architectures) and one that does not (and only has one architecture).
this change is superfluous